configure

```diff
@@ -144,13 +144,13 @@ fi
 # and then move it.
 # Note that the resulting chplconfig is expected to cover the same
 # material as any existing chplconfig.
-./util/printchplenv --overrides --anonymize > configured-chplconfig
+./util/printchplenv --simple --no-tidy --overrides --anonymize > configured-chplconfig
 mv configured-chplconfig chplconfig
 
 echo
 echo "  Currently selected Chapel configuration:"
 echo
-./util/printchplenv --anonymize
+./util/printchplenv --simple --no-tidy --anonymize
```

Comment:
I don't think this should add `--simple` - we want it to be "pretty". Additionally, I'm not convinced --no-tidy is helpful here.